lib/docusign_rest/client.rb

```diff
@@ -345,7 +345,12 @@ def get_signers(signers, options={})
         end
 
         if signer[:email_notification]
-          doc_signer[:emailNotification] = signer[:email_notification]
```

Comment:
if we already have a node for this, why do we need to modify the logic here?

Reply:
and if we have to make changes, then can we not simply use transform keys to change to camel case?

Reply:
I think we were not using this node earlier as we haven't added `email_notification` key while building signers on Manager side. Now @shamsultech89 using this key while building signer. Generally, we were following ruby style for naming keys on Manager side and here we are checking and assigning it to camel case.